api/samples/bbv.cpp

```diff
@@ -0,0 +1,342 @@
+/* **********************************************************
```

Comment:
I think this should be a first-class DR tool in clients/drbbv/.  api/samples are smaller, less polished, meant more as starting points for users to modify than final tools, and isolated with no auxiliary support.  drbbv OTOH is meant to be used as-is with end to end support for feeding to post processing frameworks.

Reply:
Right now we're not planning additional files or post-processing tools here in the dynamorio repo (internally we will process the basic block vectors generated by this client though).
There might be additional files/tools in the future.

Still ok with me to move it into `clients/drpoints` though.
Or does your opinion change if it's only one file?
Perhaps we should just call this the BBV client and DrPoints the processed basic block vectors (into instruction intervals), since those are the actual "points"?

Reply:
Moved to `client/drpoints`.